### ISSUE TITLE
Any idea about opencv problem?

### ISSUE THREAD

**Issue by hwu84** [dmlc/mxnet#1470]
Hi, following the link
https://no2147483647.wordpress.com/2016/01/16/setup-amazon-aws-gpu-instance-with-mxnet/
I have successfully installed CUDA and all the command before "make -j8". When I used the make command, some error related to opencv happened. I have also tried to make without opencv by changing the config.mk file. However it did not fix it. Can anyone help me ? Thanks!!

Attached is the screenshot of the error: 
<img width="958" alt="screenshot 2016-02-14 16 10 26" src="https://cloud.githubusercontent.com/assets/13645961/13037020/2fe3012c-d33c-11e5-9cf3-1379e2868a10.png">

**Comment by piiswrong** [comment]
You probably didn't install opencv properly.

**Comment by hwu84** [comment]
@piiswrong Thank you so much for the quick reply. I rebuilt it again and the opencv problem seems disappeared.  But when I followed the R package installation:

Rscript -e "install.packages('devtools', repo = 'https://cran.rstudio.com')"
cd R-package
Rscript -e "library(devtools); library(methods); options(repos=c(CRAN='https://cran.rstudio.com')); install_deps(dependencies = TRUE)"
cd ..
make rpkg
R CMD INSTALL mxnet_0.5.tar.gz

The following error occured. Any idea about this?  I appreciated your help. Thanks!
![photo 16-2-14 6 04 25](https://cloud.githubusercontent.com/assets/13645961/13037445/6a736dc2-d346-11e5-83c4-fe03853e8e55.jpg)

![photo 16-2-14 6 06 39](https://cloud.githubusercontent.com/assets/13645961/13037446/6ef62bfa-d346-11e5-8864-63844ef99c5a.jpg)
